README.md

```diff
@@ -253,3 +253,21 @@ Exposes [superagent](https://visionmedia.github.io/superagent/) with no modifica
 
 [travis-url]: https://travis-ci.org/staeco/vandelay-util
 [travis-image]: https://travis-ci.org/staeco/vandelay-util.png?branch=master
+
+## Development
+
+### Running Tests
+
+In order to execute tests against network services, ensure the environment variable `VANDELAY_UTIL_CONFIG` contains the path to a javascript or json file with the required configuration.
+
+You can do so by adding the following to your `~/.bashrc`:
+
+```sh
+export VANDELAY_UTIL_CONFIG="/home/gilbates/git/van-config/config.js"
```

Comment:
what is with gilbates?

Reply:
I don't like adding files to the home directory (or anywhere outside the repo), having a .env in the repo folder file would be much easier